common/common_helper.sh

```diff
@@ -294,3 +350,13 @@ function check_minikube() {
 	fi
 }
 
+###########################################
+#   Check if kind is installed
+###########################################
+function check_kind() {
```

Comment:
If check_kind fails to find Kind then Ask a prompt to install if yes then install

Reply:
As discussed, since this will require `sudo` permissions, we won't be giving the option to the user to automatically install it.